central/processlisteningonport/store/postgres/store.go

```diff
@@ -0,0 +1,748 @@
+// Code generated by pg-bindings generator.
```

Comment:
nit:  `// Code originally generated by pg-bindings generator.`

Reply:
My preference would be to do it now and get it over with so it doesn't linger.  It also clears it up that the store is in fact no longer generated.  But again that is my opinion.

Reply:
I have to disagree here. Having this done will introduce unnecessary divergence between generated version and generated bits of the custom version, making it much more painful to reconcile them in the future. I mean, how silly would that be when we will get conflicts just because of `podId` will be different from `podID`. And the fast that the store is not longer generated is stated clearly everywhere else.

If you have a strong opinion on this topic, I would be fine following your suggestion, but will recruit you in the future for reconciliation work :)

Reply:
One thing to note, such divergence is already unavoidable as #4210 resolved a lot of the style issues for these generated stores.  that specifically targeted things like changing podId to be podID.  of course an option could be to save off the custom bits, regenerate with #4210 and then add the custom stuff back.